src/Bahyph.xml

```diff
@@ -0,0 +1,29 @@
+<SPDX name="Bahyph License" identifier="Bahyph" osi-approved="false">
+  <urls>
+    <url>https://fedoraproject.org/wiki/Licensing/Bahyph</url>
+  </urls>
+  <license>
+    <optional>
+      <p>COPYRIGHT NOTICE</p>
```

Comment:
The copyright tag is intended to surround an actual copyright notice, so this falls more under the matching rules about omittable text